include/openssl/hpke.h

```diff
@@ -0,0 +1,466 @@
+/*
+ * Copyright 2019-2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/**
+ * @file
+ * APIs and data structures for HPKE (RFC9180).
+ */
+
+#ifndef OSSL_HPKE_H
+# define OSSL_HPKE_H
+# pragma once
+
+# include <openssl/ssl.h>
+
+# ifdef __cplusplus
+extern "C" {
+# endif
+
+/*
+ * The HPKE modes
+ */
+# define OSSL_HPKE_MODE_BASE              0 /**< Base mode  */
+# define OSSL_HPKE_MODE_PSK               1 /**< Pre-shared key mode */
+# define OSSL_HPKE_MODE_AUTH              2 /**< Authenticated mode */
+# define OSSL_HPKE_MODE_PSKAUTH           3 /**< PSK+authenticated mode */
+
+/*
+ * The (16bit) HPKE algorithn IDs
+ */
+# define OSSL_HPKE_KEM_ID_RESERVED         0x0000 /**< not used */
+# define OSSL_HPKE_KEM_ID_P256             0x0010 /**< NIST P-256 */
+# define OSSL_HPKE_KEM_ID_P384             0x0011 /**< NIST P-256 */
```

Comment:
Use P-384 in comment

Reply:
argh, missed that, will fix in next push

Reply:
done